elastic/datadog_checks/elastic/metrics.py

```diff
@@ -734,61 +770,25 @@ def stats_for_version(version, jvm_rate=False):
         metrics.update(JVM_METRICS_PRE_0_90_10)
 
     # Additional Stats metrics
-    if version >= [0, 90, 5]:
-        metrics.update(ADDITIONAL_METRICS_POST_0_90_5)
-    else:
+    if version < [0, 90, 5]:
```

Comment:
nit - this is not needed anymore as the first version we test is `1.1.0`, you can get rid of this `if`

Reply:
True, but we haven't yet officially dropped support for 0.90.5, we're just not testing for it. 🤷 

I have a green light from Shri to drop this version, will do it in a separate PR and tag him and Anjali.

Reply:
Not saying we should drop the `if` completely, just saying we can always add the metrics to the list 🙂